clients/spring-boot-starter-camunda-sdk/src/test/java/io/camunda/spring/client/properties/CamundaClientPropertiesPostProcessorTest.java

```diff
@@ -58,6 +59,30 @@ void shouldMapLists() {
     }
   }
 
+  @Nested
```

Comment:
These test should go to `JavaClientPropertiesTest` as these tests focus on the actual properties handling.

`CamundaClientPropertiesPostProcessorTest` performs assertions on the post processor behaviour:

* activating modes
* mapping legacy properties